lib/ansible/plugins/action/iosxr.py

```diff
@@ -48,7 +48,17 @@ def run(self, tmp=None, task_vars=None):
         elif self._play_context.connection == 'local':
             provider = load_provider(iosxr_provider_spec, self._task.args)
             pc = copy.deepcopy(self._play_context)
-            pc.connection = 'network_cli'
+            if self._task.action in ['iosxr_netconf', 'iosxr_config', 'iosxr_command'] or \
```

Comment:
iosxr_config and iosxr_command should support both netconf and cli?

Reply:
iosxr does not support cli commands over netconf. Keeping support to take xml rpc schema as input at later priority over individual config modules.